tests/fbchat_test.py

```diff
@@ -0,0 +1,240 @@
+# test fbchat api
+import sys
+from unittest.mock import patch
+# from postr import config
+# import fbchat_api
+from postr import fbchat_api
+sys.path.insert(0, '../postr')
+
+
+class Object():
+    name: str = ''
+    uid: str = ''
+    first_name: str = ''
+    last_name: str = ''
+    text: str = ''
+
+
+# password: str = str(config.get_api_key('TESTFB', 'password'))
+# email: str = str(config.get_api_key('TESTFB', 'email'))
+
+client = fbchat_api.FacebookChatApi('ddo3@case.edu', 'seniorproject')
```

Comment:
should be removed from the code

Reply:
That is commented out because the config file with my values cant be pushed to the repo.  Whenever my tests tried to get these values, it would fail

Reply:
Yep, we can't/shouldn't push actual passwords/usernames to the repo, in the code or in the config. All of our tests have to work without actual private API access

Reply:
So do you want me to just remove those lines?

Reply:
yep, or you can also explicitly only enable the tests when the code is running locally, for example, see "Skipping Tests that Hit the Real API" in https://realpython.com/testing-third-party-apis-with-mock-servers/

Reply:
But only use that for tests that absolutely positively cannot be fully mocked